docs/tasks/administer-federation/cluster.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Federated Cluster
+redirect_from:
+- "/docs/user-guide/federation/cluster/"
+- "/docs/user-guide/federation/cluster.html"
+---
+
+{% capture overview %}
+
+This guide explains how to use Clusters in a Federation control plane.
+
+Different than other Kuberentes resources, such as Deployments, Services and ConfigMaps,
+clusters only exist in the federation context, i.e. those requests must be submitted to the
+federation api-server.
+
+{% endcapture %}
+
+{% capture prerequisites %}
+
+* {% include federated-task-tutorial-prereqs.md %}
+* You should also have a basic [working knowledge of Kubernetes](/docs/setup/pick-right-solution/) in
+general.
+
+{% endcapture %}
+
+{% capture steps %}
+
+## Listing Clusters
+
+To list the clusters available in your federation, you can use [kubectl](/docs/user-guide/kubectl/) by
+running:
+
+``` shell
+kubectl --context=federation-cluster get clusters
```

Comment:
We are trying to avoid using `federation-cluster` as a context in other docs as well. See new `kubefed` docs for example. Can we just say `federation` or `my-federation` or something like that?